rcl/test/rcl/test_graph.cpp

```diff
@@ -706,6 +706,80 @@ TEST_F(
   rcl_reset_error();
 }
 
+/* Test the rcl_count_clients function.
+ *
+ * This does not test content of the response.
```

Comment:
Can we add the test case to check the count with creating service? I think it would be really nice to check these APIs with CI to make sure all Tier 1 implementation returns expected result actually.

this test does not have to be in `rcl` but probably `rmw` or `rmw_implementation`.

Reply:
added in `rmw_implentation`
https://github.com/ros2/rmw_implementation/pull/208/files#diff-c2432e189d173195f3c5a57637a628cfab56b07960c07d66992f21dc5cd88478R977